src/main/resources/static/stylesheets/style.css

```diff
@@ -49,4 +49,16 @@
 
 .navbar-option {
     box-shadow: 0 0 1px 0 #33332a;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this usage of the experimental "box-shadow" property. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=css%3Aexperimental-property-usage)